client/src/ExtensionConstants.ts

```diff
@@ -30,4 +30,8 @@ export class ExtensionConstants {
     public static readonly REFRESH_SERVICES = "extension.refreshServices";
     public static readonly REFRESH_SERVICES_VIEW_ID = "service-providers";
     public static readonly NODE_TLS_REJECT_UNAUTHORIZED = "NODE_TLS_REJECT_UNAUTHORIZED";
-}
\ No newline at end of file
+
+    public static readonly VS_CODE_MESSAGE_COMMAND_LOGIN= "login";
```

Comment:
Let's add a space before the `=` sign.

Reply:
https://github.com/wso2-extensions/identity-tools-plugin-vscode/pull/7